invoice/src/main/java/org/killbill/billing/invoice/config/MultiTenantInvoiceConfig.java

```diff
@@ -223,14 +223,30 @@ public UsageDetailMode getItemResultBehaviorMode(final InternalTenantContext ten
         if (result != null){
             return UsageDetailMode.valueOf(result);
         }
-
         if (mode == UsageDetailMode.AGGREGATE || mode == UsageDetailMode.DETAIL) {
             return mode;
         }
-
         return UsageDetailMode.AGGREGATE;
     }
 
+    @Override
+    public AccountTzOffset getAccountTzOffsetMode() {
+        final AccountTzOffset mode = staticConfig.getAccountTzOffsetMode();
+        if (mode == AccountTzOffset.FIXED || mode == AccountTzOffset.VARIABLE) {
```

Comment:
I see we're doing this in other enum configs as well. Do you remember why? Don't we just need a null check?

Reply:
I don 't think this is really required, I was trying to be consistent -  I suspect because we start from a 'string' we want double check it is one of the expected value and if not default.